File: Projects/server/Contracts/Requests/UpdatePersonInfoRequest.cs
namespace server.Contracts.Requests
{
    public class UpdatePersonInfoRequest
    {
        public string Email { get; set; } = null!;
        public string Phone { get; set; } = null!;
        public string LastName { get; set; } = null!;
        public string FirstName { get; set; } = null!;
        public string? Patronymic { get; set; }
        public DateOnly Birth { get; set; }
        public string? DriveLicense { get; set; }
    }
}

File: Projects/DataProviderApp/app/src/main/java/com/example/dataproviderapp/dto/requests/UpdatePersonInfoRequest.kt
package com.example.dataproviderapp.dto.requests

import java.time.LocalDate

data class UpdatePersonInfoRequest(
    val email: String,
    val phone: String,
    val lastName: String,
    val firstName: String,
    val patronymic: String?,
    val birth: LocalDate,
    val driveLicense: String?
)
